### DISCUSSION TITLE
[Mini.files] I experience lag when opening mini.files.

### DISCUSSION THREAD

**Discussion by LintaoAmons** [nvim-mini/mini.nvim#1163] [General]
![mini files](https://github.com/user-attachments/assets/c2eb26ed-37a5-4394-a7eb-06e954318926)

After I press my keybinding, it takes almost 3 seconds for the program to open,
while NeoTree shows and locates the file almost instantly.

**Comment by echasnovski** [comment]
Unfortunately, I don't think I can reproduce:

https://github.com/user-attachments/assets/b0bedfe7-2219-4beb-9a6d-a8aa02bad62d

Does delay go away if you disable preview? Also, does delay go away if (*with* preview) you disable tree-sitter highlighting in 'typescript' filetype?

**Reply by LintaoAmons** [reply]
My bad...
I have another keybinding start with `<leader>e`
All good now~ Thanks!
